tracking/imu/imu_lsm6dso.h

```diff
@@ -0,0 +1,23 @@
+#ifndef IMU_LSM6DSO_H   /* Include guard */
+#define IMU_LSM6DSO_H
+
+#include "../../lib/lsm6dso-api/lsm6dso_reg.h"
+
+#include <furi_hal.h>
+
+#include "imu.h"
+
+#define LSM6DSO_TAG "LSM6DO-C"
+
+#define LSM6DSO_DEV_ADDRESS UINT8_C(0xD6)
```

Comment:
Nit: for uniformity, it's probably better to rename it to `LSM6DSO_ADDRESS` and specify address as `(0x6B << 1)`.

Reply:
(Actually, renaming `LSM6DS3_ADDRESS` to `LSM6DS3_DEV_ADDRESS` might be more uniform.)